static/css/mkp.scss

```diff
@@ -13,9 +21,16 @@ header .intro-text {
     .intro-subtext {
         font-family: {{ site.data.template.font.secondary }};
     }
-
 }
 
+.btn-xl {
+	background-color: #{{ site.data.template.color.buttonyellow }};
+	color: #fff;
+	font-weight: bold;
+	font-size: 150%;
+	text-shadow: 0 0 2px #440, 0 0 20px #440;
```

Comment:
I wonder if we should sync all text-shadows to 440

Reply:
I used this color here because the background is yellow. But removing, since it's not in the Figma.